mozilla-mobile/android-components/components/feature/autofill/src/main/res/values/strings.xml

```diff
@@ -13,4 +13,22 @@
     <!-- Autofill: Text shown in popup in third-party app to autofill the password for an account.
     %1$s will be replaced with the login/username of the account. -->
     <string name="mozac_feature_autofill_popup_password">Password for %1$s</string>
+
+    <!-- Autofill: Title of a dialog asking the user to confirm before autofilling credentials into
+    a third-party app after the authenticity verification failed. -->
+    <string name="mozac_feature_autofill_confirmation_title">Verification failed</string>
+
+    <!-- Autofill: Text shown in dialog asking the user to confirm before autofilling credentials into a
+    third-party app where we could not verify the authenticity (e.g. we determined that this app is
+    a twitter client and we could autofill twitter credentials, but according to the "Digital Asset
+    Links" this application is not the official Twitter application for twitter.com credentials. -->
+    <string name="mozac_feature_autofill_confirmation_authenticity">%1$s could not verify the authenticity of the application. Do you want to proceed with autofilling the selected credentials?</string>
```

Comment:
@pocmo Could you please add what the placeholder will be replaced here with, to give context to localizers? I can expose everything on Pontoon once that is done. Thanks!